.changeset/olive-bears-act.md

```diff
@@ -0,0 +1,107 @@
+---
+'@primer/react': major
+---
+
+![FormControl component using a label, text input, validation and hint](https://user-images.githubusercontent.com/13340707/154948443-60d70cc7-4f26-444f-8366-7963336be53c.png)
+
+The `FormControl` component will be used to deprecate the `FormGroup`, `InputField`, and `ChoiceInputField` components. It has the ability to render contextual content with your inputs: labels, validation statuses, captions. It also handles the ARIA attributes that make the form controls accessible to assistive technology.
+
+<table>
+<tr>
+<th> Before </th> <th> After </th>
+</tr>
+<tr>
+<td valign="top">
+
+```jsx
+import {FormControl, Checkbox, TextInput} from "@primer/react"
+
+
+<FormGroup>
+    <FormGroup.Label htmlFor="example-text">Example text</FormGroup.Label>
+    <TextInput id="example-text" />
+</FormGroup>
+
+OR
```

Comment:
syntax highlighting looks a bit off in the snippet, might be this. could we turn this into a comment?